src/lambda.rs

```diff
@@ -0,0 +1,94 @@
+mod commp;
+
+use std::error::Error;
+use std::str::FromStr;
+
+#[macro_use]
+extern crate lambda_runtime as lambda;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate log;
+extern crate simple_logger;
```

Comment:
Not needed, just remove it as you cal use top-level modules without importing them in the Rust 2018 edition.